documentation/src/test/java/org/hibernate/userguide/mapping/generated/GeneratedTest.java

```diff
@@ -24,7 +24,7 @@
 /**
  * @author Vlad Mihalcea
  */
-@RequiresDialect(value = SQLServer2005Dialect.class, majorVersion = 9)
+@RequiresDialect(value = SQLServerDialect.class, majorVersion = 9)
```

Comment:
Since we raise the minimum version to 10, you can remove the `majorVersion` arguments when <= 10. Not sure if you want to do this as part of this PR or separately, but that would be a nice cleanup.

```suggestion
@RequiresDialect(value = SQLServerDialect.class)
```